waku/waku_core/message/digest.nim

```diff
@@ -11,6 +11,11 @@ import ../topics, ./message
 
 type WakuMessageHash* = array[32, byte]
 
+const EmptyWakuMessageHash*: WakuMessageHash = [
+  0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+  0, 0, 0,
+]
```

Comment:
Maybe this can work too:
```suggestion
const EmptyWakuMessageHash*: WakuMessageHash = newSeqWith(32, 0.byte)
```

Reply:
`WakuMessageHash` is an `array` not a `seq`. I've not found a way to init arrays with multiple values sadly.